RadClass/H0.py

```diff
@@ -55,10 +68,18 @@ def run_gross(self, data, timestamp):
             self.x2 = data
             n = self.x1 + self.x2
             p = 0.5
-            pval = stats.binom_test(self.x1, n, p, alternative='two-sided')
-
-            # only save instances with rejected null hypothesesf
-            if pval <= self.significance:
+            # scipy.stats.binomtest will fail if n (# of trials)
+            # is less than 1 (possible for low count-rate integration times)
+            if int(n) < 1:
+                pval = 0.0
+            else:
+                pval = np.log10(stats.binomtest(int(self.x1), int(n), p,
+                                                alternative='two-sided').pvalue)
+                if np.isinf(pval):
+                    pval = -350
```

Comment:
I recommend a small private method that does this.  It's only used twice so far, but it you want to tweak things in a consistent way, it's helpful for it to be in one place.

Reply:
You can also get rid of the magic number and name it `min_log_pval` for context